package.json

```diff
@@ -7,6 +7,7 @@
   "jsnext:main": "build/three.module.js",
   "module": "build/three.module.js",
```

Comment:
It'll be a problem if the `module` entry is `build/three.module.js` but the JSM example files all import from `src/Three.js`. In this case...

```js
import { Vector3 } from 'three';
import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader.js';
```

... GLTFLoader will create Vector3 instances that are not the same as the Vector3 instances from `three`.

Reply:
Nice catch!

My objective was to find a solution for people who already import the main three.js from the src/ folder for the tree-shaking to take effect, like this:

```js
import { Vector3 } from 'three/src/math/Vector3';
import { GLTFLoader } from 'three/examples/jsm/loaders/GLTFLoader.js';
```

Currently if they do that for the `examples/jsm` the whole bundle will be imported as well.

Is there a way we can achieve this in your opinion?